packages/components/src/navigator/index.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * Internal dependencies
+ */
+import { Navigator as TopLevelNavigator } from './navigator';
+import { NavigatorScreen } from './navigator-screen';
+import { NavigatorButton } from './navigator-button';
+import { NavigatorBackButton } from './navigator-back-button';
+export { useNavigator } from './use-navigator';
+
+/**
+ * The `Navigator` component allows rendering nested views/panels/menus
+ * (via the `Navigator.Screen` component) and navigate between these different
+ * view (via the `Navigator.Button` and `Navigator.BackButton` components or the
+ * `useNavigator` hook).
+ *
+ * @example
```

Comment:
```suggestion
```

Should we remove this `@example` tag so the snippet shows up in the Storybook?

Reply:
https://github.com/WordPress/gutenberg/pull/64613/commits/e7d20195ce125e78a2594874445bb34cb939766d